merkle_zeppelin/data_io/json_importer.py

```diff
@@ -0,0 +1,8 @@
+from .base_importer import MerkleTreeImporter
+from .dto import MerkleTreeDTO
+
+
+class MerkleTreeJSONImporter(MerkleTreeImporter):
+    @staticmethod
+    def import_tree(data: str) -> MerkleTreeDTO:
+        return MerkleTreeDTO.model_validate(data)
```

Comment:
pydantic `model_validate` and input type str ? 🤔

Reply:
here you have point - should be `model_validate_json`